monitoring-alert-rules/README.md

```diff
@@ -25,36 +26,37 @@ You need access to the `kyma-system` Namespace to execute the described steps.
     kubectl port-forward pod/prometheus-monitoring-0 -n kyma-system 9090:9090
     ```
 
-    Find the **http-db-service-is-not-running** rule [here](http://localhost:9090/rules).
+3. Go to `http://localhost:9090/rules` and find the **pod-not-running** rule.
 
-    As the `http-db-service` Deployment does not the exist, the alert is fired [here](http://localhost:9090/alerts).
+    Because the http-db-service Deployment does not the exist, [Alertmanager](https://kyma-project.io/docs/components/monitoring#details-alertmanager) fires an alert listed at `http://localhost:9090/alerts`.
```

Comment:
```suggestion
    Because the `http-db-service` Deployment does not exist, [Alertmanager](https://kyma-project.io/docs/components/monitoring#details-alertmanager) fires an alert listed at `http://localhost:9090/alerts`.
```

Reply:
Done :)